main.css

```diff
@@ -602,3 +602,225 @@ ul {
+  font-family: Poppins, sans-serif;
+  font-weight: 400;
+  font-style: normal;
+  line-height: 24px;
+  margin-top: 12px;
+  order: 1;
+}
+* */
+
+/* Duplicated from line: 550
+ *
+.resume__button {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 12px;
+  gap: 10px;
+  background-color: #fff;
+  margin-top: 22px;
+  border-radius: 8px;
+  border: 1px solid #5f6ef1;
+  width: 130px;
+  height: 40px;
+}
+* */
+
+/* Duplicated from line: 569
+ *
+.resume__coding {
+  display: flex;
+  margin-top: 36px;
+  background-color: #f6f6ff;
+  width: 303px;
+  height: 66px;
+  margin-left: 16px;
+  border-radius: 8px;
+}
+* */
+
+/* Duplicated from line: 589
+.resume__coding img {
+  display: flex;
+  margin-top: 8px;
+  margin-left: 10px;
+  width: 40px;
+  height: 38px;
+}
+* */
+
+/* Duplicated from line: 597
+ *
+.framework__container {
+  display: flex;
+  margin-top: 70px;
+  color: #172b4d;
+  height: 20px;
+  width: 10px;
+  font-weight: 600;
+}
+* */
+
+.contact__container_me {
+  position: absolute;
+  width: cover;
+  left: 1px;
+  background: #6070ff;
+  border-top-left-radius: 80px;
+}
+
+.contact__name {
+  position: absolute;
+  background: #fff;
+}
+
+.Contact__me {
+  display: flex;
+  min-width: 327px;
+  height: 56px;
+  flex-direction: column;
+  justify-content: center;
+  flex-shrink: 0;
+  color: #fff;
+  text-align: center;
+  font-family: var(--FF-primary-poppins);
+  font-size: 40px;
+  font-style: normal;
+  font-weight: 700;
+  line-height: 52px;
+}
+
+.Contact__text {
+  min-width: 327px;
+  height: 137px;
+  flex-shrink: 0;
+  color: #ebebff;
+  text-align: center;
+  font-family: var(--FF-primary-poppins);
+  font-size: 20px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 28px; /* 140% */
+  padding: 12px 24px 12px 24px;
+}
+
+input {
+  display: block;
+  border-radius: 8px;
+  border: 1px solid #cfd8dc;
+  background: #fff;
+  min-width: 327px;
+  height: 48px;
+  margin-top: 20px;
+  margin-left: 24px;
+  padding: 15px 38px 15px 16px;
+}
+
+textarea {
+  display: block;
+  border-radius: 8px;
+  border: 1px solid #cfd8dc;
+  background: #fff;
+  margin-top: 18px;
+  min-width: 327px;
+  height: 170px;
+  margin-left: 24px;
+  padding: 15px 38px 15px 16px;
+}
+
+.contact__button {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 12px;
+  gap: 10px;
+  background-color: white;
+  border: 1px solid #6070ff;
+  width: 162px;
+  height: 48px;
+  border-radius: 8px;
+  cursor: pointer;
+  margin-top: 22px;
+  margin-left: 24px;
+}
+
+.contact__info {
+  background-image: url(assets/contact-form.png);
+  padding: 90px 24px;
+  border-top-left-radius: 90px;
+  background-size: contain, cover;
+  background-position: right;
+  background-clip: padding-box, padding-box;
+  background-repeat: no-repeat, no-repeat;
+  display: flex;
+  flex-direction: column;
+  text-align: center;
+  align-items: stretch;
+  justify-content: center;
+  color: #fff;
+}
```

Comment:
:red_circle: [REQUIRED]  I noticed that this section is not centered correctly. Please center the section and attach the background as it is on the [Figma Design.](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=39-122&mode=design) :arrow_down: 

|FIgma|Your design|
|:-------:|:--------------:|
|![good](https://github.com/AsciencioAlex/Mobile-Finish-Portfolio/assets/26736582/5f8843b3-5e3d-4f03-bd3d-a39ca9ced98c)|![ur design](https://github.com/AsciencioAlex/Mobile-Finish-Portfolio/assets/26736582/e934a6d4-7266-42b7-90e5-085f9078f6c7)|